pkg/operator/ceph/disruption/clusterdisruption/reconcile.go

```diff
@@ -34,8 +34,6 @@ import (
 	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
 	"github.com/rook/rook/pkg/operator/ceph/disruption/controllerconfig"
 	"github.com/rook/rook/pkg/operator/k8sutil"
-	policyv1 "k8s.io/api/policy/v1"
-	policyv1beta1 "k8s.io/api/policy/v1beta1"
```

Comment:
LGTM, the linter just has an error about the unused var

Reply:
Yup, removed the unused variable.